documentation/source/newsfragments/1411.feature.rst

```diff
@@ -0,0 +1,4 @@
+Custom logging handlers can now access the simulator time using
+``LogRecord.created_sim_time``, provided the
+:class:`~cocotb.log.SimTimeContextFilter` filter is not removed from the logger
+instance.
```

Comment:
That describes the internal change well. How about also describing the user-visible outcome of this, i.e. time stamps are now matching the time when the log entry was created, and not when it was printed.

Reply:
I don't think this is actually user visible unless the user is doing something very unusual.